services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategy.java

```diff
@@ -58,12 +55,12 @@ public void validateForEdus(ApiToken apiToken) {
    */
   public void validateForPpa(ApiToken apiToken) {
     apiToken.getLastUsedPpac().ifPresent(getLastUsedEpochSecond -> {
-      LocalDateTime currentTimeUtc = LocalDateTime.now(clock);
+      LocalDateTime currentTimeUtc = TimeUtils.getLocalDateTimeForNow();
       LocalDateTime lastUsedForPpaUtc = ofEpochSecond(getLastUsedEpochSecond).atOffset(UTC).toLocalDateTime();
-      long hours = HOURS.between(lastUsedForPpaUtc, currentTimeUtc);
-      if (hours < VALIDITY_IN_HOURS) {
-        logger.info("Api Token was updated {} hours ago. Api Token can only be used once every {} hours.",
-            hours, VALIDITY_IN_HOURS);
+      long seconds = SECONDS.between(lastUsedForPpaUtc, currentTimeUtc);
+      if (seconds < validityInSeconds) {
+        logger.info("Api Token was updated {} seconds ago. Api Token can only be used once every {} seconds.",
```

Comment:
Even if the check itself is in seconds. For better readability we maybe should still stick to a more concrete timestamp instead of seconds?

Reply:
Hmm. I know it's less readable, but I would keep the logging the same logic and values as the actual check, to not hide any lurking errors... or how would you actually do it?

Reply:
Ok - understandable. I mean we can just extract hours:minutes:seconds from just seconds. So the log would be actually "Api Token was updated 23:56:10 hours ago ..." or "Api Token was updated 23h 56min 10secs ago...."

Reply:
Good idea. Will do.